html/components/masthead.stories.js

```diff
@@ -369,28 +369,29 @@ export const defaultStory = () => {
               class="sprk-c-MastheadAccordion__summary"
               href="#nogo"
             >
-                <span
+              <span
+                class="
+                  sprk-b-TypeBodyOne
+                  sprk-c-MastheadAccordion__heading
+                "
+              >
+                <svg
                   class="
-                    sprk-b-TypeBodyOne
-                    sprk-c-MastheadAccordion__heading
+                    sprk-c-Icon
+                    sprk-c-Icon--xl
+                    sprk-c-Icon--filled-current-color
+                    sprk-c-Icon--stroke-current-color
+                    sprk-u-mrs
                   "
+                  viewBox="0 0 64 64"
                 >
-                  <svg
-                    class="
```

Comment:
These are spacing fixes, github is showing them strangely in the unified view. Its easier to look in split view for this.

Reply:
hahah it's VERY confusing for this PR!